lstchain/tools/lstchain_create_dl3_file.py

```diff
@@ -194,6 +194,9 @@ def start(self):
                 f'{self.energy_dependent_gh_cuts.meta["GH_EFF"]}'
             )
         else:
+            self.cuts.global_gh_cut = QTable.read(
```

Comment:
Again, there is no need to read the full table to access a single header object.

Reply:
Ah right! I will fix it.